index.html

```diff
@@ -191,62 +197,178 @@ <h2>Multi-Post Stories</h2>
+            <img src="./images/Counter.svg" alt="" class="counter" />
+            <span class="text-light">2018</span>
+          </div>
+          <p>
+            A smart assistant to make driving more safe, efficient, and fun by
+            unlocking your most expensive computer: your car.
+          </p>
+          <ul class="categories">
+            <li>html</li>
+            <li>Ruby on rails</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="btn card-button">See Project</button>
+        </div>
+        <img
+          class="card-image"
+          src="./images/availability.svg"
+          alt="A template with some forms and an image of a woman"
+        />
+      </article>
     </section>
 
     <!-- About-Section -->
     <section class="about-section">
       <div class="about-page">
-        <h2 class="about-header">About<br />Myself</h2>
-        <p>
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestitate
-          to contact me.
-        </p>
-        <h3 class="connect-text">LET'S CONNECT</h3>
-        <ul class="headline-icons">
-          <li>
-            <a href=""
-              ><img
-                src="./images/twitter.svg"
-                alt="A twitter logo"
-                class="headline-icon"
-            /></a>
-          </li>
-          <li>
-            <a href=""
-              ><img
-                src="./images/Linkedin.svg"
-                alt="A github logo"
-                class="headline-icon"
-            /></a>
-          </li>
-          <li>
-            <a href=""
-              ><img
-                src="./images/m-logo.svg"
-                alt="A m-logo"
-                class="headline-icon"
-            /></a>
-          </li>
-          <li>
-            <a href=""
-              ><img
-                src="./images/github.svg"
-                alt="A github logo"
-                class="headline-icon"
-            /></a>
-          </li>
-          <li>
-            <a href=""
-              ><img
-                src="./images/peace-sign.svg"
-                alt="A peace sign logo"
-                class="headline-icon"
-            /></a>
-          </li>
-        </ul>
-        <button class="about-button">Get my resume</button>
+        <div class="about-container">
+          <h2 class="about-header">About<br />Myself</h2>
+          <p class="about-paragraph">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestitate to contact me.
+          </p>
+          <h3 class="connect-text">LET'S CONNECT</h3>
+          <ul class="headline-icons">
```

Comment:
[OPTIONAL] This is only optional because it was not mentioned by the previous reviewer. Making sure your design looks as close to the Figma template as possible is part of the project requirements. I strongly suggest that you take this into consideration.

Kindly increase the height of your about page in order to increase the `top` and `bottom`  padding like in the Figma design:

Your page:
![image](https://user-images.githubusercontent.com/38088818/165926434-37893110-87d3-44ec-9846-82285c62acde.png)

Figma:
![image](https://user-images.githubusercontent.com/38088818/165927665-eeb7bf7e-4e84-4c4d-aa9c-0f4332ef55d4.png)

I made these changes to your desktop media query and it did the trick. This is the same height as the Figma template, you're welcome to try it:
![image](https://user-images.githubusercontent.com/38088818/165928759-df2eb3c4-5710-41d1-9b3e-14faaca036cf.png)